pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
-    * `pip install poetry`
-* [Git](https://git-scm.com/downloads)
-    * [Windows](https://git-scm.com/download/win)
-    * [MacOS](https://git-scm.com/download/mac) or `brew install git`
-    * [Linux](https://git-scm.com/download/linux)
-* A running webserver for the [site](../site)
-    * Follow the linked guide only if you don't want to use Docker or if you plan to do development on the site project too.
-
-## Using Docker
-
-Both the site and the bot can be started using Docker.
-Using Docker is generally recommended (but not strictly required) because it abstracts away some additional set up work, especially for the site.
-However, if you plan to attach a debugger to either the site or the bot, run the respective project directly on your system (AKA the _host_) instead.
-
-The requirements for Docker are:
-
-* [Docker CE](https://docs.docker.com/install/)
-* [Docker Compose](https://docs.docker.com/compose/install/) (This already comes bundled on macOS and Windows, so you shouldn't need to install it)
-    * `pip install docker-compose`
-
----
-# Fork the project
-You will need access to a copy of the git repository of your own that will allow you to edit the code and push your commits to.
-Creating a copy of a repository under your own account is called a _fork_.
-
-* [Learn how to create a fork of the repository here.](../forking-repository)
-
-This is where all your changes and commits will be pushed to, and from where your PRs will originate from.
-
-For any staff member, since you have write permissions already to the original repository, you can just create a feature branch to push your commits to instead.
-
----
-# Development environment
-1. [Clone your fork to a local project directory](../cloning-repository/)
-2. [Install the project's dependencies](../installing-project-dependencies/)
-3. [Prepare your hosts file (Optional)](../hosts-file/)
-
----
-# Test server and bot account
-You will need your own test server and bot account on Discord to test your changes to the bot.
-
-* [**Create a test server**](../setting-test-server-and-bot-account#setting-up-a-test-server)
-* [**Create a bot account**](../setting-test-server-and-bot-account#setting-up-a-bot-account)
-* Invite it to the server you just created.
-
-### Privileged Intents
-
-With `discord.py` 1.5 and later, it is now necessary to explicitly request that your Discord bot receives certain gateway events.
+The purpose of this guide is to get you a running local version of [the Python bot](https://github.com/python-discord/bot).
+This page will focus on the quickest steps one can take, with mentions of alternatives afterwards.
+
+### Clone The Repository
+First things first, to run the bot's code and make changes to it, you need a local version of it (on your computer).
+
+Clone the repository from [here](https://github.com/python-discord/bot). If you are not a member of the organisation, you will need to create a fork of the project, and clone the fork instead.
```

Comment:
I think we can specify what a `member of the organisation` is, since that may not be clear for people new to Github.
Minor comment though, feel free to take it or leave it.

Reply:
Reworded that section, is that better?